tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -506,13 +506,22 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
     }
 
     // Only include the memory and timeout limit if set
-    if flags.action.memory > 0 || flags.action.timeout > 0 {
+    if flags.action.memory > -1 || flags.action.timeout > -1 || flags.action.logsize > -1 {
         limits = new(whisk.Limits)
-        if flags.action.memory > 0 {
-            limits.Memory = flags.action.memory
+        if flags.action.memory > -1 {
+            limits.Memory = &flags.action.memory
+        } else {
+            limits.Memory = nil
```

Comment:
Allocated pointers are initialized to nil, so there's actually no need to explicitly set these limits.* pointers to nil.